src/components/ModalVote.vue

```diff
@@ -86,8 +86,8 @@ async function handleSubmit() {
     emit('openPostVoteModal');
     emit('reload');
     addVotedProposalId(props.proposal.id);
-    emit('close');
   }
+  emit('close');
```

Comment:
Will close the vote modal even if it is not a success?

Reply:
Yes, this is because we want to show the other modal. The way the modal component is setup it will not look good if they are stacked with different heights.

Reply:
I mean if the hub returns some other error other than the wrong timestamp error

with this change this happens:
 

[Tab-1676099282762.webm](https://user-images.githubusercontent.com/15967809/218245713-031e52d0-54be-42f5-a6bb-7c4c55b808dc.webm)

Reply:
Right good catch! I will create another issue for this